build-farm/sign-sbom.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Use this example as the basis of what this script needs to do: https://github.com/adoptium/temurin-build/blob/master/build-farm/sign-releases.sh